### PR TITLE
Feature/core/create bom

### PR DESCRIPTION
**Pull request by TalhaAli00**

### CODE REVIEW

File: androidCore/bom/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="com.walletconnect.android.bom" />
```
Comment (Elyniss) [inline androidCore/bom/src/main/AndroidManifest.xml:2]: Might not be true but we can probably get rid of this file if we add `namespace` property to build.gradle

```
android {
	namespace = "com.walletconnect.android.bom"
}
```

File: sign/wallet/build.gradle.kts [hunk 2]
```diff
@@ -44,10 +44,13 @@ dependencies {
     implementation(project(":sign:samples_common"))
 
     debugImplementation(project(":sign:sdk"))
-    releaseImplementation("com.walletconnect:sign:2.2.0")
+//    releaseImplementation("com.walletconnect:sign:2.2.0")
 
     debugImplementation(project(":androidCore:sdk"))
-    releaseImplementation("com.walletconnect:android-core:1.4.0")
```
Comment (Elyniss) [inline sign/wallet/build.gradle.kts:50]: should we add bom to other project `releaseImplementation`s as well?

File: androidCore/sdk/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.walletconnect.android.api">
```
Comment (Elyniss) [inline androidCore/sdk/src/main/AndroidManifest.xml:3]: nice catch!

File: androidCore/bom/src/main/AndroidManifest.xml (hunk 1 above)
Comment (TalhaAli00) [reply androidCore/bom/src/main/AndroidManifest.xml]: removed the manifest

File: sign/wallet/build.gradle.kts (hunk 2 above)
Comment (TalhaAli00) [reply sign/wallet/build.gradle.kts:50]: updated all of the samples to use bom. BOM will need to be built locally until we release

**Review by jakobuid — APPROVED** [review]

**Pull request merged**